.github/workflows/ci.yml

```diff
@@ -55,6 +55,47 @@ jobs:
         find $HOME/.cache/coursier/v1                -name "ivydata-*.properties" -delete || true
         find $HOME/.sbt                              -name "*.lock"               -delete || true
 
+  js-test:
```

Comment:
I think we need to either adjust the name so that it's consistent with others or update mergify's config